helm/istio/charts/tracing/templates/jaeger-production-elasticsearch.yaml

```diff
@@ -12,7 +12,9 @@ spec:
   strategy: production
 
   query:
+    {{- if and .Values.jaeger.hub .Values.jaeger.tag }}
     image: {{ .Values.jaeger.hub }}/jaeger-query:{{ .Values.jaeger.tag }}
```

Comment:
The image name should be jaeger-query-rhel7, @objectiser can you please confirm?